cmd/web-handlers.go

```diff
@@ -32,9 +32,9 @@ import (
 	"strings"
 	"time"
 
+	"archive/zip"
 	"github.com/gorilla/mux"
 	"github.com/gorilla/rpc/v2/json2"
-	"github.com/klauspost/compress/zip"
```

Comment:
Why this change?

Reply:
I suppose you need this: https://github.com/klauspost/compress/pull/200

Reply:
BTW we also need to fix the #8712

Reply:
```
diff --git a/cmd/web-handlers.go b/cmd/web-handlers.go
index 4ef80a6d..c102b4e8 100644
--- a/cmd/web-handlers.go
+++ b/cmd/web-handlers.go
@@ -1403,6 +1403,8 @@ func (web *webAPIHandlers) DownloadZip(w http.ResponseWriter, r *http.Request) {
                        header := &zip.FileHeader{
                                Name:   strings.TrimPrefix(objectName, args.Prefix),
                                Method: zip.Deflate,
+                               Flags:  1 << 11,
+                               Modified: info.ModTime,
                        }
                        if hasStringSuffixInSlice(info.Name, standardExcludeCompressExtensions) || hasPattern(standardExcludeCompressContentTypes, info.ContentType) {
                                // We strictly disable compression for standard extensions/content-types.
diff --git a/go.mod b/go.mod
index 0ff2752f..816d036b 100644
--- a/go.mod
+++ b/go.mod
@@ -32,7 +32,7 @@ require (
        github.com/hashicorp/vault/api v1.0.4
        github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
        github.com/json-iterator/go v1.1.7
-       github.com/klauspost/compress v1.8.3
+       github.com/klauspost/compress v1.9.5
        github.com/klauspost/pgzip v1.2.1
        github.com/klauspost/readahead v1.3.1
        github.com/klauspost/reedsolomon v1.9.3
diff --git a/go.sum b/go.sum
index 1c7413c5..c1682b5e 100644
--- a/go.sum
+++ b/go.sum
@@ -480,6 +480,8 @@ github.com/klauspost/compress v1.5.0/go.mod h1:RyIbtBH6LamlWaDj8nUwkbUhJ87Yi3uG0
 github.com/klauspost/compress v1.8.2/go.mod h1:RyIbtBH6LamlWaDj8nUwkbUhJ87Yi3uG0guNDohfE1A=
 github.com/klauspost/compress v1.8.3 h1:CkLseiEYMM/fRb0RIg9mXB+Iwgmle+U9KGFu+JCO4Ec=
 github.com/klauspost/compress v1.8.3/go.mod h1:RyIbtBH6LamlWaDj8nUwkbUhJ87Yi3uG0guNDohfE1A=
+github.com/klauspost/compress v1.9.5 h1:U+CaK85mrNNb4k8BNOfgJtJ/gr6kswUCFj6miSzVC6M=
+github.com/klauspost/compress v1.9.5/go.mod h1:RyIbtBH6LamlWaDj8nUwkbUhJ87Yi3uG0guNDohfE1A=
 github.com/klauspost/cpuid v0.0.0-20160106104451-349c67577817/go.mod h1:Pj4uuM528wm8OyEC2QMXAi2YiTZ96dNQPGgoMS4s3ek=
 github.com/klauspost/cpuid v1.2.0/go.mod h1:Pj4uuM528wm8OyEC2QMXAi2YiTZ96dNQPGgoMS4s3ek=
 github.com/klauspost/cpuid v1.2.1 h1:vJi+O/nMdFt0vqm8NZBI6wzALWdA2X+egi0ogNyrC/w=

```

Reply:
`v1.9.6` is released.